lib/server/routes/health.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const Router = require('./index');
+const inherits = require('util').inherits;
+
+function HealthRouter(options) {
+  if(!(this instanceof HealthRouter)) {
+    return new HealthRouter(options);
+  }
+
+  Router.apply(this, arguments);
+}
+
+inherits(HealthRouter, Router);
+
+HealthRouter.prototype.health = function(req, res) {
+  if (!this.storage) {
```

Comment:
We should probably check `this.storage.connection.readyState == 1` to make sure there is a connection.